old/index.html

```diff
@@ -0,0 +1,30 @@
+<head>
```

Comment:
Olease remove all unused files (the entire `old/` directory can go)

Reply:
done